implementations/rust/ockam/ockam_api/src/kafka/secure_channel_map.rs

```diff
@@ -155,10 +155,10 @@ impl<F: RelayCreator> Clone for KafkaSecureChannelControllerImpl<F> {
 }
 
 /// Describe to reach the consumer node:
-/// either directly or through a relay with a relay
+/// either directly or through a relay
 #[derive(Clone)]
 pub(crate) enum ConsumerNodeAddr {
-    Direct(Option<MultiAddr>),
+    Direct(MultiAddr),
```

Comment:
In direct mode, you can take the two roles, consumer or producer, the producer will specify the consumer in the `consumer_route` argument, but the consumer won't have anything to specify (unless both nodes have both roles).
Maybe we can add a third `None` option?

Reply:
Okey, I didn't know that. I think adding a third option will make things more understandable.